guides/common/modules/proc_updating-server.adoc

```diff
@@ -0,0 +1,70 @@
+[id="Updating_Server_{context}"]
+= Updating {ProjectServer}
+
+ifdef::satellite[]
+Use this procedure to update your connected {ProjectServer} to the next minor version.
+For information to update a disconnected {Project} setup, see xref:Updating-Disconnected-{project-context}_{context}[].
+endif::[]
+ifndef::satellite[]
+Use this procedure to update your {ProjectServer} to the next minor version.
+endif::[]
+
+.Prerequisites
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+ifndef::satellite[]
+* Ensure that you have synchronized repositories for {ProjectServer} and {SmartProxyServers} to {Project}.
+endif::[]
+ifdef::satellite[]
+* Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
++
+** {RepoRHEL8BaseOS}
+** {RepoRHEL8AppStream}
+** {RepoRHEL8ServerSatelliteServerProductVersion}
+** {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+endif::[]
```

Comment:
According to [this comment](https://github.com/theforeman/foreman-documentation/pull/2233#discussion_r1279834294), this isn't required. Remove.